### PR TITLE
Use glob in /scripts

### LINKED ISSUE #239: Move scripts to Node.js `glob`
In many scripts of `scripts/` and `web/scripts/` we use a lot of JS code to walk through the folders.

We can make code simpler by moving to glob. In Node.js there is native glob so we don’t need library for that.

### PR DESCRIPTION
**Pull request by 3timeslazy**
Fixes #239

### CODE REVIEW

**Comment by ai** [comment]
CI is broken to fix it:

1. Run `eslint --fix .` to sort variables
2. Add `fs.globSync` to `ignore` of `n/no-unsupported-features/node-builtins` rule in ESLint config

File: scripts/check-versions.ts [hunk 1]
```diff
@@ -71,14 +71,8 @@ if (packageManager !== pnpmFull) {
...
-    let relative = join(project, 'package.json')
-    checkPackage(relative, read(relative))
-  }
-  if (existsSync(join(ROOT, project, 'Dockerfile'))) {
-    let docker = join(project, 'Dockerfile')
-    checkDockerfile(docker, read(docker))
-  }
+let files = globSync(['./*/package.json', '**/Dockerfile'])
+for (let file of files) {
+  if (file.endsWith('package.json')) checkPackage(file, read(file))
```
Comment (ai) [inline scripts/check-versions.ts:76]: Let’s do 2 `globSync` calls here like

```js
for (let file of globSync('./*/package.json')) {
  checkPackage(file, read(file))
}
```

File: eslint.config.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ export default [
     }
   },
   {
-    files: ['web/pages/**/*', 'web/ui/**/*', 'web/main/**/*', 'core/test/**/*'],
+    files: ['web/pages/**/*', 'web/ui/**/*', 'web/main/**/*', 'core/test/**/*', 'scripts/*'],
```
Comment (ai) [inline eslint.config.js:21]: We disable this rule only for code running browser.

We can disable it for the whole project or make an allow-list of experimental API for Node.js related code like scripts.

Comment (3timeslazy) [reply eslint.config.js:21]: Would that work?

```js
{
    files: ['scripts/**.ts'],
    rules: {
      'n/no-unsupported-features/node-builtins': ['error', {
          ignores: ['fs.globSync']
      }]
    }
  }
```

**Comment by 3timeslazy** [comment]
@ai btw, it seems like `check-version` does not check Dockerfile versions. It matches against `node: (\d+\.\d+\.\d+)` pattern while Dockerfiles have `NODE_VERSION (\d+\.\d+\.\d+)` pattern

**Comment by ai** [comment]
Yes, this is issue.

Can you fix it here if in separated PR?

**Comment by 3timeslazy** [comment]
> Yes, this is issue.
> 
> Can you fix it here if in separated PR?

I'll fix it in a separate PR than

File: scripts/check-versions.ts (hunk 1 above)
Comment (3timeslazy) [reply scripts/check-versions.ts]: Done

File: eslint.config.js (hunk 2 above)
Comment (3timeslazy) [reply eslint.config.js]: Done

**Comment by ai** [comment]
Thanks!

**Pull request merged**